lib/node_modules/@stdlib/math/base/napi/ternary/README.md

```diff
@@ -182,6 +182,45 @@ The function accepts the following arguments:
 void stdlib_math_base_napi_fff_f( napi_env env, napi_callback_info info, float (*fcn)( float, float, float ) );
 ```
 
+#### stdlib_math_base_napi_dii_d( env, info, fcn )
+
+Invokes a ternary function accepting a double-precision floating-point number and two signed 32-bit integers and returning a double-precision floating-point number.
+
+```c
+#include <node_api.h>
+
+// ...
+
+static double add( const double x, const int32_t y, const int32_t z ) {
+    return x + y + z;
+}
+
+// ...
+
+/**
+* Receives JavaScript callback invocation data.
+*
+* @param env    environment under which the function is invoked
+* @param info   callback data
+* @return       Node-API value
+*/
+napi_value addon( napi_env env, napi_callback_info info ) {
+    return stdlib_math_base_napi_dii_d( env, info, add );
+}
```

Comment:
```suggestion
static double fcn( const double x, const int32_t y, const int32_t z ) {
    // ...
}

// ...

/**
* Receives JavaScript callback invocation data.
*
* @param env    environment under which the function is invoked
* @param info   callback data
* @return       Node-API value
*/
napi_value addon( napi_env env, napi_callback_info info ) {
    return stdlib_math_base_napi_dii_d( env, info, fcn );
}
```

We can make the example function more opaque in this case, as adding two 32-bit ints to a double is a bit odd.